src/components/Create/index.tsx

```diff
@@ -454,13 +454,13 @@ export default function Create() {
 
           {buildSteps([
             {
-              title: 'Appearance',
+              title: 'Project details',
               description: 'Project name, handle, links, and other details.',
               callback: () => setProjectFormModalVisible(true),
             },
             {
               title: 'Funding',
-              description: 'How your project will earn and manage funds.',
+              description: "Your project's funding cycle target and duration.",
```

Comment:
Loosely held opinion. I think there needs to be mention of funding cycle, ideally.

Reply:
I agree, a nice and consistent repetition of JB vocab seems like a good strategy